tests/click_tests/reset.py

```diff
@@ -0,0 +1,72 @@
+import shamir_mnemonic as shamir
+
+from trezorlib import messages
+
+from .. import buttons
+
+
+def confirm_wait(debug, startswith):
+    layout = debug.wait_layout()
+    assert layout.text.startswith(startswith)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def confirm_read(debug, startswith):
+    layout = debug.read_layout()
+    assert layout.text.startswith(startswith)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def set_selection(debug, button, diff):
+    layout = debug.read_layout()
+    assert layout.text.startswith("Slip39NumInput")
+    for _ in range(diff):
+        debug.click(button, wait=False)
+    layout = debug.click(buttons.OK, wait=True)
+
+
+def words_read(debug, is_advanced=False):
+    # read words
```

Comment:
Maybe we can just call the function `read_words`? :)

Reply:
ok I renamed it, I also renamed words_confirm to confirm_words to stay consistent.